src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -555,6 +569,13 @@ def get(self, args):
         # Check the command index for (command: [module]) mapping, like
         # "network": ["azure.cli.command_modules.natgateway", "azure.cli.command_modules.network", "azext_firewall"]
         index_modules_extensions = index.get(top_command)
+        if not index_modules_extensions and self.cli_ctx.data['completer_active']:
+            # If user type `az acco`, command begin with `acco` will be matched.
+            logger.debug("In autocomplete mode, load command begin with: '%s'", top_command)
```

Comment:
```suggestion
            logger.debug("In autocomplete mode, load commands starting with: '%s'", top_command)
```

Let's be consistent with Python's naming `startswith`: https://docs.python.org/3/library/stdtypes.html#str.startswith